packit_service/service/events.py

```diff
@@ -188,7 +188,9 @@ def __init__(self, trigger: TheJobTriggerType, project_url: str):
         )
 
     def get_project(self) -> GitProject:
-        return ServiceConfig.get_service_config().get_project(url=self.project_url)
+        project = ServiceConfig.get_service_config().get_project(url=self.project_url)
+        logger.debug(f"GitService = {project.service.__dict__}")
```

Comment:
This is very dangerous -- print the `str` representation instead so we don't log service secrets.

Reply:
dangerous where? that's literally the point here: print the secrets so we know what's the setup

I mean, I can drop that commit since the issue is resolved - it's just these log lines helped me resolve it

thinking... it's true that we share srpm logs with people and this could potentially be printed in there, I'll drop it

Reply:
The `str` version contains the first and last character of the tokens to be sure we are using the right tokens...